test/unit/active_model/serializer/associations_test.rb

```diff
@@ -14,6 +14,36 @@ def test_associations_inheritance
         assert_equal([:comments],
                      another_inherited_serializer_klass._associations.keys)
       end
+      def test_multiple_nested_associations
+        parent = SelfReferencingUserParent.new(name: "The Parent")
```

Comment:
Ah my bad! Always get confused between 0.9 and 0.10. Then I guess it's better to keep it the way it is to stay consistent with 0.9 way of testing.